spec/requests/requests_requests_spec.rb

```diff
@@ -48,6 +48,34 @@
           expect(response).to have_http_status(:not_found)
         end
       end
+
+      context 'When organization has a default storage location' do
+        let(:request) { create(:request, organization: create(:organization, default_storage_location: 1)) }
+        it 'shows the column Default storage location inventory' do
+          get request_path(request)
+
+          expect(response.body).to include('Default storage location inventory')
+        end
+      end
+
+      context 'When partner has a default storage location' do
+        let(:storage_location) { create(:storage_location) }
+        let(:request) { create(:request, partner: create(:partner, default_storage_location_id: storage_location.id)) }
+        it 'shows the column Default storage location inventory' do
+          get request_path(request)
+
+          expect(response.body).to include('Default storage location inventory')
+        end
+      end
+
+      context 'When partner or organization does not have a default storage location' do
```

Comment:
This is a nitpick, but this should be 'When neither partner nor organization has a default storage location' -- i.e. that it has to be both the partner doesn't and the organization doesn't.   Your description means, instead,  if the partner doesn't have it or the organization doesn't have it,   which is not quite the same thing.

Reply:
Fixed it here [90098cd](https://github.com/rubyforgood/human-essentials/pull/4466/commits/90098cda2a4c7f659ea771b588e5ab0b830257c0)